mods/webui/src/stories/overviewcard/OverviewCard.stories.ts

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (C) 2024 by Fonoster Inc (https://fonoster.com)
+ * http://github.com/fonoster/fonoster
+ *
+ * This file is part of Fonoster
+ *
+ * Licensed under the MIT License (the "License");
+ * you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    https://opensource.org/licenses/MIT
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { Meta, StoryObj } from "@storybook/react";
+import { fn } from "@storybook/test";
+import { OverviewCard } from "./OverviewCard";
+import { Icon } from "../icon/Icon";
+import React from "react";
+
+/**
+ * This story is for the OverviewCard component based on MUI switch component
```

Comment:
Would you update the comment since the component is custom, as opposed to MUI base?

Reply:
okay